index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Josh Lee</title>
+
+  <!-- Leave these links here...
+  But this should be in a comment... -->
+  <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link type="text/css" rel="stylesheet" href="style.css"/>
+</head>
+<body>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="app__content">
```

Comment:
Here You are using the `app__content` but the only `app__content` is later on in the page.

This was labeled as `abc` and I think you have that replacement correct on the next line where you replaced `abc` with `top-nav`.